Factorial/Factorial.c

```diff
@@ -0,0 +1,35 @@
+/* This code is used to find factorial 
+of large numbers*/
+#include<stdio.h>
+int main()
+{
+    int n,a[200000],i,j,k,l,m;
+    printf("ENTER THE NUMBER\n");
+	scanf("%d",&n);
```

Comment:
keep a space after comma and correct the indentation

Reply:
Done. pls check it out @asha15